{"name": "test_01[args0]", "status": "broken", "statusDetails": {"message": "requests.exceptions.InvalidURL: Failed to parse: http://127.0.0.1:5000http://120.26.10.118:2256", "trace": "self = <PreparedRequest [GET]>\nurl = 'http://127.0.0.1:5000http://120.26.10.118:2256', params = OrderedDict()\n\n    def prepare_url(self, url, params):\n        \"\"\"Prepares the given HTTP URL.\"\"\"\n        #: Accept objects that have string representations.\n        #: We're unable to blindly call unicode/str functions\n        #: as this will include the bytestring indicator (b'')\n        #: on python 3.x.\n        #: https://github.com/psf/requests/pull/2238\n        if isinstance(url, bytes):\n            url = url.decode('utf8')\n        else:\n            url = unicode(url) if is_py2 else str(url)\n    \n        # Remove leading whitespaces from url\n        url = url.lstrip()\n    \n        # Don't do any URL preparation for non-HTTP schemes like `mailto`,\n        # `data` etc to work around exceptions from `url_parse`, which\n        # handles RFC 3986 only.\n        if ':' in url and not url.lower().startswith('http'):\n            self.url = url\n            return\n    \n        # Support for unicode domain names and paths.\n        try:\n>           scheme, auth, host, port, path, query, fragment = parse_url(url)\n\n/Users/mr.xiao/anaconda3/envs/ienv/lib/python3.7/site-packages/requests/models.py:382: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nurl = 'http://127.0.0.1:5000http://120.26.10.118:2256'\n\n    def parse_url(url):\n        \"\"\"\n        Given a url, return a parsed :class:`.Url` namedtuple. Best-effort is\n        performed to parse incomplete urls. Fields not provided will be None.\n        This parser is RFC 3986 compliant.\n    \n        The parser logic and helper functions are based heavily on\n        work done in the ``rfc3986`` module.\n    \n        :param str url: URL to parse into a :class:`.Url` namedtuple.\n    \n        Partly backwards-compatible with :mod:`urlparse`.\n    \n        Example::\n    \n            >>> parse_url('http://google.com/mail/')\n            Url(scheme='http', host='google.com', port=None, path='/mail/', ...)\n            >>> parse_url('google.com:80')\n            Url(scheme=None, host='google.com', port=80, path=None, ...)\n            >>> parse_url('/foo?bar')\n            Url(scheme=None, host=None, port=None, path='/foo', query='bar', ...)\n        \"\"\"\n        if not url:\n            # Empty\n            return Url()\n    \n        source_url = url\n        if not SCHEME_RE.search(url):\n            url = \"//\" + url\n    \n        try:\n            scheme, authority, path, query, fragment = URI_RE.match(url).groups()\n            normalize_uri = scheme is None or scheme.lower() in NORMALIZABLE_SCHEMES\n    \n            if scheme:\n                scheme = scheme.lower()\n    \n            if authority:\n                auth, _, host_port = authority.rpartition(\"@\")\n                auth = auth or None\n                host, port = _HOST_PORT_RE.match(host_port).groups()\n                if auth and normalize_uri:\n                    auth = _encode_invalid_chars(auth, USERINFO_CHARS)\n                if port == \"\":\n                    port = None\n            else:\n                auth, host, port = None, None, None\n    \n            if port is not None:\n                port = int(port)\n                if not (0 <= port <= 65535):\n                    raise LocationParseError(url)\n    \n            host = _normalize_host(host, scheme)\n    \n            if normalize_uri and path:\n                path = _remove_path_dot_segments(path)\n                path = _encode_invalid_chars(path, PATH_CHARS)\n            if normalize_uri and query:\n                query = _encode_invalid_chars(query, QUERY_CHARS)\n            if normalize_uri and fragment:\n                fragment = _encode_invalid_chars(fragment, FRAGMENT_CHARS)\n    \n        except (ValueError, AttributeError):\n>           return six.raise_from(LocationParseError(source_url), None)\n\n/Users/mr.xiao/anaconda3/envs/ienv/lib/python3.7/site-packages/urllib3/util/url.py:394: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nvalue = None, from_value = None\n\n>   ???\nE   urllib3.exceptions.LocationParseError: Failed to parse: http://127.0.0.1:5000http://120.26.10.118:2256\n\n<string>:3: LocationParseError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <testcase.test_api.TestApi object at 0x7f9b96719e90>\nargs = {'api_name': '获取FITMORE分类标签', 'api_request': {'headers': {'Content-type': 'application/json'}, 'method': 'post', 'parmas': {'training_platform': '8'}, 'url': 'http://120.26.10.118:2256'}, 'api_validate': [{'eq': {'code': 200}}]}\n\n    @pytest.mark.parametrize('args', read_yaml_data(\"traning.yaml\"))\n    def test_01(self, args):\n        url = args['api_request']['url']\n        method = args['api_request']['method']\n        headers = args['api_request']['headers']\n        parmas = args['api_request']['parmas']\n>       res = self.httpRequest.send_request(method=method, url=url, headers=headers, data=parmas)\n\ntest_api.py:20: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../utils/ReqUtil.py:63: in send_request\n    res = self._post(self.last_url, self.last_headers, self.last_data, files)\n../utils/ReqUtil.py:27: in _post\n    return requests.get(url=url, headers=headers, data=data, files=files)\n/Users/mr.xiao/anaconda3/envs/ienv/lib/python3.7/site-packages/requests/api.py:76: in get\n    return request('get', url, params=params, **kwargs)\n/Users/mr.xiao/anaconda3/envs/ienv/lib/python3.7/site-packages/requests/api.py:61: in request\n    return session.request(method=method, url=url, **kwargs)\n/Users/mr.xiao/anaconda3/envs/ienv/lib/python3.7/site-packages/requests/sessions.py:528: in request\n    prep = self.prepare_request(req)\n/Users/mr.xiao/anaconda3/envs/ienv/lib/python3.7/site-packages/requests/sessions.py:466: in prepare_request\n    hooks=merge_hooks(request.hooks, self.hooks),\n/Users/mr.xiao/anaconda3/envs/ienv/lib/python3.7/site-packages/requests/models.py:316: in prepare\n    self.prepare_url(url, params)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <PreparedRequest [GET]>\nurl = 'http://127.0.0.1:5000http://120.26.10.118:2256', params = OrderedDict()\n\n    def prepare_url(self, url, params):\n        \"\"\"Prepares the given HTTP URL.\"\"\"\n        #: Accept objects that have string representations.\n        #: We're unable to blindly call unicode/str functions\n        #: as this will include the bytestring indicator (b'')\n        #: on python 3.x.\n        #: https://github.com/psf/requests/pull/2238\n        if isinstance(url, bytes):\n            url = url.decode('utf8')\n        else:\n            url = unicode(url) if is_py2 else str(url)\n    \n        # Remove leading whitespaces from url\n        url = url.lstrip()\n    \n        # Don't do any URL preparation for non-HTTP schemes like `mailto`,\n        # `data` etc to work around exceptions from `url_parse`, which\n        # handles RFC 3986 only.\n        if ':' in url and not url.lower().startswith('http'):\n            self.url = url\n            return\n    \n        # Support for unicode domain names and paths.\n        try:\n            scheme, auth, host, port, path, query, fragment = parse_url(url)\n        except LocationParseError as e:\n>           raise InvalidURL(*e.args)\nE           requests.exceptions.InvalidURL: Failed to parse: http://127.0.0.1:5000http://120.26.10.118:2256\n\n/Users/mr.xiao/anaconda3/envs/ienv/lib/python3.7/site-packages/requests/models.py:384: InvalidURL"}, "attachments": [{"name": "log", "source": "fe4ebcb3-2264-4186-a4b8-8a26a9191d0d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "args", "value": "{'api_name': '获取FITMORE分类标签', 'api_request': {'url': 'http://120.26.10.118:2256', 'method': 'post', 'headers': {'Content-type': 'application/json'}, 'parmas': {'training_platform': 8}}, 'api_validate': [{'eq': {'code': 200}}]}"}], "start": 1626402133629, "stop": 1626402133631, "uuid": "0f76df1c-df1d-489a-ad29-39c57d8a6f78", "historyId": "905bdc911946778aebb87d79ebe313cc", "testCaseId": "9e95fb0ee651629b1d2a91d4c46bcf77", "fullName": "testcase.test_api.TestApi#test_01", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "MrdeMacBook-Pro.local"}, {"name": "thread", "value": "36824-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_api"}]}